### DISCUSSION TITLE
My application cannot be reviewed.

### DISCUSSION THREAD

**Discussion by Rhynd** [community/community#57036] [GitHub Education]
### Hi everyone,

Hello, I tried to get the GitHub student application, but it keeps telling me that I need a bio, that I have, and my full name as on my documents, that I have too, but I still can't apply. And now I need to wait a few days because it kepps refusing my name.... 
Help me please :'(
![image_2023-06-02_190450005](https://github.com/community/community/assets/135255682/bdd18099-26cd-4177-9daa-c9097c35a28d)

**Comment by J0J0HA** [comment]
You have to use your full name as stated on the document you uploaded as GitHub display name, which seems not to be the case.

**Reply by Rhynd** [reply]
I used the same name as on the documents, I tried with my last name first, and my first name first, but no matter which is first, it's still wrong...

**Comment by samus-aran** [comment]
Hi @JustRhynd 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
